.travis/utils.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
 # SPDX-License-Identifier: MIT
```

Comment:
This not new added, but why the license is MIT? Any hint @tyll?

Reply:
@nhosoi please remove this file from the commit if it has no changes

Reply:
??? All the changes made on utils.sh or just this `# SPDX-License-Identifier: MIT` line? If former, I had to fix some shellcheck errors in the file.

Reply:
Since we have established that files copied from `template/` can retain their MIT license even though the rest of `network/` is licensed under BSD-3, then we don't need to change the license in this file.
And hopefully this will all go away soon . . . https://github.com/linux-system-roles/tox-lsr

Reply:
Anyway, I reverted utils.sh since the changes are not needed as we skip shellcheck for network.